src/App.tsx

```diff
@@ -1,8 +1,78 @@
-import React from 'react';
+import React, { FC, useState } from 'react';
 import './App.css';
+import { TodoList } from './components/TodoList/TodoList';
+import { TodoType, UserType, TodoWithUsers } from './types';
+import { getUsers, getTodos } from './api';
 
-const App = () => (
-  <h1>Dynamic list of TODOs</h1>
-);
+const App: FC<{}> = () => {
+  const [todos, setTodos] = useState<TodoWithUsers[]>([]);
+  const [isLoading, setLoading] = useState<boolean>(false);
+  const [filteredTodos, setFilteredTodos] = useState<TodoWithUsers[]>([...todos]);
```

Comment:
давай логически сортировать
сначала идут тудусы, потом условные значения

также для примитивов не нужно указывать тип, они автоматически появтяся
```suggestion
  const [todos, setTodos] = useState<TodoWithUsers[]>([]);
  const [filteredTodos, setFilteredTodos] = useState<TodoWithUsers[]>([...todos]);
  const [isLoading, setLoading] = useState(false);
```

Reply:
Ок